### PR TITLE
Build Tools .Net Core 3

### PR DESCRIPTION
**Pull request by HurricanKai**
# Summary of the PR
Updates Build Tools to .Net Core 3
Resolving any Issues

# Related issues, Discord discussions, or proposals
See #41 for further information

# Further Comments

### CODE REVIEW

**Comment by HurricanKai** [comment]
Not clear why Azure Pipelines is failing. Seems to be the nuget feed?
Else ready to review.

**Comment by HurricanKai** [comment]
There we go. Pipelines works.

**Comment by Perksey** [comment]
Have you tried a full run of the BuildTools?
```
./generator.bat build/binder_scripts/fullrun.json
```

**Comment by HurricanKai** [comment]
Yep, I did. Set it up as my Run config in Rider.

File: src/Core/BuildTools/GLXmlConvert/Construction/ParsingHelpers.cs [hunk 1]
```diff
@@ -263,22 +259,44 @@ public static Count ParseCountSignature
                 return new Count(valueReferenceName);
             }
 
-            // Some counts are a value reference along with some sort of numerical operation - let's skip forward in
-            // the string until we hit the first invalid character, and see if the rest are just numbers or math ops
-            var dataBeforeFirstInvalidCharacter = new string(countData.TakeWhile(char.IsLetterOrDigit).ToArray());
```
Comment (Perksey) [inline src/Core/BuildTools/GLXmlConvert/Construction/ParsingHelpers.cs]: We should reinstate these 2 lines as I liked the if statements we had before. Split the parsing logic into its own function, and then do `if (SyntaxFacts.IsValidIdentifier(dataBeforeFirstInvalidCharacter) && IsMath(dataAfterFirstInvalidCharater)`

Comment (HurricanKai) [reply src/Core/BuildTools/GLXmlConvert/Construction/ParsingHelpers.cs]: Added a small commit to do this, does that work for you?

Comment (Perksey) [reply src/Core/BuildTools/GLXmlConvert/Construction/ParsingHelpers.cs]: Yes that works nicely :) open another PR as I've already merged this one :P

**Pull request merged**